source/_components/notify.webostv.markdown

```diff
@@ -49,3 +50,14 @@ automation:
       data:
         message: "You should open a window! (Livingroom Co2: {{ states.sensor.netatmo_livingroom_co2.state }}ppm)"
 ```
+
+The icon can be overridden for individual notifications by providing a path to an alternative icon image to use:
+
+```yaml
```

Comment:
Since it's fairly common for folks to copy the examples I think you should make it a full automation example as the one above.